core/src/main/kotlin/org/evomaster/core/problem/rest/service/ResourceRestModule.kt

```diff
@@ -76,5 +77,9 @@ class ResourceRestModule : AbstractModule(){
 
         bind(TestSuiteWriter::class.java)
                 .asEagerSingleton()
+
+        bind(object: TypeLiteral<ExternalServices>() {})
```

Comment:
to simplify, it could be `bind(ExternalServices::class.java)
                .asEagerSingleton()`